.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: IRC Channel
+    url: https://web.libera.chat/#vis-editor
```

Comment:
IRC URL should be `ircs://irc.libera.chat:6697/vis-editor`, shouldn’t it?

Reply:
I'm not sure its that important. For the people who actively use IRC I think "`#vis-editor` on Libera" is sufficient without the link.